Doc/library/math.rst

```diff
@@ -199,8 +199,8 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return ``e**x``.
-
+   Return e raised to the power *x*, where e = 2.718281... is the base
```

Comment:
It would be best to use Sphinx's [inline math](http://www.sphinx-doc.org/en/stable/ext/math.html#role-math) construct here.  Is there anything that would prevent using `:math:` role in Python documentation?